src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double longerBase;
+    private final double shorterBase;
+    private final double height;
+
+    public IsoscelesTrapezoid(double longerBase, double shorterBase, double height, String color) {
+        super(color);
+        this.longerBase = longerBase;
+        this.shorterBase = shorterBase;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((shorterBase + longerBase) * height) / 2;
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: isosceles trapezoid, area: " + getArea() + " sq"
```

Comment:
Use 'name()' for getting String representation of enum constants instead of 'toString()' when printing the color.